Orange/tests/test_evaluation_testing.py

```diff
@@ -191,6 +192,11 @@ def test_continuous(self):
             self.housing, [LinearRegressionLearner()])
         self.assertLess(RMSE(res), 5)
 
+    def test_mape_percentage(self):
+        res = CrossValidation(k=3)(
+            self.housing, [LinearRegressionLearner()])
+        self.assertGreater(MAPE(res), 1)
```

Comment:
Not necessarily true in general, but apparently true for housing. :)

Reply:
Absolutely, however, it is very difficult to test whether something is truly in the range of 0-100 and not 0-1. :)